src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -22,22 +18,11 @@ import { UNIT } from 'Core/Geographic/GeoCoordinate';
  * @param {String} options.format: image format (default: format/jpeg)
  * @returns {Object@call;create.url.url|String}
  */
-function WMS_Provider(/* options*/) {
-    // Constructor
-    Provider.call(this, new IoDriver_XBIL());
-    this.cache = CacheRessource();
-    this.projection = new Projection();
-
-    this.getTextureFloat = function getTextureFloat(buffer) {
-        // Start float to RGBA uint8
-        var texture = new THREE.DataTexture(buffer, 256, 256, THREE.AlphaFormat, THREE.FloatType);
-
-        texture.needsUpdate = true;
-        return texture;
-    };
+function WMS_Provider(options) {
+    WMTS_Provider.call(this, options);
```

Comment:
Instead of inherinting from `WMTS_Provider`, maybe we could build a helper class for all those protocols (WFS, WMS, WMTS)?